packages/code/src/Code.tsx

```diff
@@ -80,6 +92,136 @@ function getWrapperVariantStyle(variant: Variant): string {
   `;
 }
 
+function getSidebarVariantStyle(variant: Variant): string {
+  const colors = variantColors[variant];
```

Comment:
I'm not sure I understand... colors is an array, wouldn't that mean that if `variantColors[variant]` doesn't exist, then we would default to the string `'light'` rather than (I think) `variantColors['light']`?

Reply:
Ah, you're right! So fallback to `Variant.Light`.

My basic idea was to make sure we always have a fallback if we don't have an empty state.